validation/validator.go

```diff
@@ -51,12 +51,16 @@ type failedValidator struct {
 
 // IsStringInSlice tells if a searched value is part of a slice or not
 func IsStringInSlice(slice []string, searched string) bool {
-	for _, value := range slice {
-		if value == searched {
-			return true
-		}
+	return slices.Contains(slice, searched)
+}
```

Comment:
Then is this function needed anymore?

Reply:
I did not wanted to remove it now... but for sure, now it exists in golang, we should use slice.Contains(...)